### PR TITLE
Create README_RU.md

### PR DESCRIPTION
**Pull request by Barkov-F**

### CODE REVIEW

File: 0-DeveloperEnvironment/README_RU.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+Часть ответа ноды появляется в том же терминале, где вы запускали код. Другая часть там, где вы запускали ноду, т.е. в первом терминале. Обязательно проверяйте оба окна, пока не освоитесь, где появляется какая часть. 
+
+## Cryptofex IDE
+Pyrofex готовится выпустить среду разработки приложений на платформах RChain и Etherium под названием [cryptofex](https://cryptofex.io/). Сейчас она находится в alpha стадии. При помощи этого IDE можно создавать не только dApp для RChain, но и смартконтракты для EVM.
+
+## Плагин для IntelliJ IDEA
```
Comment (JoshOrndorff) [inline 0-DeveloperEnvironment/README_RU.md]: Can you also add this to the English version?

Comment (JoshOrndorff) [reply 0-DeveloperEnvironment/README_RU.md]: @zsluedem In case you want this in the Chinese version

Comment (JoshOrndorff) [reply 0-DeveloperEnvironment/README_RU.md]: @Barkov-F I don't see the PR for the English version. Can you point me to it?

**Comment by Barkov-F** [comment]
https://github.com/JoshOrndorff/LearnRholangByExample/pull/28 I think this
is the one. Or do you want me to add the part on IntelliJ IDEA plugin for
the English version?

**Pull request merged**